app/views/nsm/steps/view_claim/work_items.html.erb

```diff
@@ -8,67 +8,76 @@
   <%= govuk_details(summary_text: t('.check_totals')) do %>
     <%=
       head = [
-        { text: t('.item'), width: 'govuk-!-width-one-third' },
+        { text: nil },
         { text: t('.time'), numeric: true },
         { text: t('.net_cost'), numeric: true },
       ]
       vat_rate = current_application.firm_office.vat_registered == 'true' ? Pricing.new(@claim).vat : 0
       total_cost = BigDecimal('0')
-      rows = @work_items.group_by(&:work_type).map do |work_type, work_items_for_type|
+      sorted_work_items = @work_items.sort_by(&:sort_position)
+      rows = sorted_work_items.group_by(&:work_type).map do |work_type, work_items_for_type|
         time = work_items_for_type.sum(&:time_spent)
         net_cost = work_items_for_type.sum(&:total_cost)
         total_cost += net_cost
         [
           { text: t("summary.nsm/cost_summary/work_items.#{work_type.to_s}"), width: 'govuk-!-width-one-quarter' },
-          { text: format_period(time), numeric: true },
+          { text: format_period(time, style: :minimal_html), numeric: true },
           { text: NumberTo.pounds(net_cost), numeric: true },
         ]
       end
 
       foot = [
         { text: t(".total"), width: 'govuk-!-width-one-quarter' },
         { text: '', numeric: true },
-        { text: NumberTo.pounds(total_cost), numeric: true },
+        { text: govuk_visually_hidden(t('.net_cost_description')) + NumberTo.pounds(total_cost), numeric: true },
       ]
 
 
       govuk_table do |table|
         table.with_head(rows: [head])
-        table.with_body(rows: rows.sort_by { _1.dig(0, :text) }, first_cell_is_header: true)
+        table.with_body(rows: rows, first_cell_is_header: true)
         table.with_foot(rows: foot)
       end
     %>
   <% end %>
 
   <%
-    pagy, work_items = pagy(@work_items)
-    head = [
-      { text: t('.item'), width: 'govuk-!-width-one-quarter' },
-      { text: t('.time'), numeric: true },
-      { text: t('.uplift'), numeric: true },
-      { text: t('.net_cost'), numeric: true },
-      { text: t('.action'), numeric: true }
-    ]
-    work_items.group_by(&:completed_on).each do |completed_on, work_items_for_date| %>
-    <%=
-      rows = work_items_for_date.map do |work_item|
-        link = link_to(t('.view'), item_nsm_steps_view_claim_path(id: current_application.id, item_type: :work_item, item_id: work_item.id, page: pagy.page), data: { turbo: 'false' })
+    pagy, work_items = pagy_array(@work_items, items: 100)
+    head = {
+      'line_item' => false,
+      'item' => false,
+      'date' => false,
+      'fee_earner' => false,
+      'time' => [],
+      'uplift' => [],
+      'net_cost' => []
+    }
+  %>
+  <%=
```

Comment:
do we need a check for no work items here
```erb
  <% if work_items.size.zero? %>
    <p><%= t('.no_data') %></p>
  <% else %>
```

Reply:
Work items are mandatory, you can't proceed the application without adding at least one work item.